docs/websocket/feedback.md

```diff
@@ -123,6 +143,16 @@ id   | `string` | **Required** The id of the feedback to pause.
 
 A paused feedback can be resumed using this RPC. After this the audio stream is
 again processed by the server.
+Currently **it is required to re-send the audio header when resuming
+feedback**. Failing to do so will fail the feedback as the server can't
+recognise the audio format without a header.
```

Comment:
I thought the server could be intelligent about the header being absent or not.
Well, if the SDK takes care of it, it's fine either way though.

Reply:
I think this feedback belongs in #73 :wink:

Reply:
it is marked as added in this PR so the feedback does belong here.
The server can't do that now, so the SDK will do it and the docs document you should do it (if you're not using the SDK).
So its fine for now indeed.

Reply:
It's only marked as added in this PR as #73 isn't merged yet. I added a comment with this PR stating "#73 Should be merged first.", I'm not saying this without reason, if you decide to ignore that comment useless discussions like this can arise.

Reply:
Solved in #73.